src/components/GradientFill.tsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react'
-import LinearGradient from 'react-native-linear-gradient'
+import {LinearGradient} from 'expo-linear-gradient'
```

Comment:
I don't think we are actually using this `GradientFill` component at all, can we RM or is this going to be used somewhere else?

Reply:
Almost used in mod services, no need to keep it around for now if ya don't want to